clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/KafkaStoreIngestionService.java

```diff
@@ -217,8 +200,6 @@ public KafkaStoreIngestionService(
       ClusterInfoProvider clusterInfoProvider,
       ReadOnlyStoreRepository metadataRepo,
       ReadOnlySchemaRepository schemaRepo,
-      Optional<CompletableFuture<HelixCustomizedViewOfflinePushRepository>> customizedViewFuture,
-      Optional<CompletableFuture<HelixInstanceConfigRepository>> helixInstanceFuture,
```

Comment:
Nitpick: I think the `sslFactory` is also unused. Can you check and delete that, if applicable?

Reply:
Done, unfortunately it's still used for KME protocol so can only be converted from class field to local variable to be passed along.